Snakefile

```diff
@@ -33,19 +39,23 @@ rule test_target:
 
 rule files:
     params:
-        dropped_strains = "config/dropped_strains_{subtype}.txt",
-        include_strains = "config/include_strains_{subtype}_{time}.txt",
-        reference = "config/reference_{subtype}_{segment}.gb",
-        colors = "config/colors_{subtype}.tsv",
-        lat_longs = "config/lat_longs_{subtype}.tsv",
+        dropped_strains = lambda w: "config/dropped_strains_{subtype}.txt" if not w.subtype=='h5n1-cattle-outbreak' else "config/dropped_strains_h5nx.txt",
+        include_strains = lambda w: "config/include_strains_{subtype}_{time}.txt" if not w.subtype=='h5n1-cattle-outbreak' else "config/include_strains_{subtype}.txt",
+        reference = lambda w: "config/reference_{subtype}_{segment}.gb" if not w.subtype=='h5n1-cattle-outbreak' else "config/reference_h5nx_{segment}.gb",
+        colors = lambda w: "config/colors_{subtype}.tsv" if not w.subtype=='h5n1-cattle-outbreak' else "config/colors_h5nx.tsv",
+        # TODO - Augur 24.4.0 includes extensive lat-longs by default - can we drop the following avian-flu specific ones?
+        lat_longs = lambda w: "config/lat_longs_{subtype}.tsv" if not w.subtype=='h5n1-cattle-outbreak' else "config/lat_longs_h5nx.tsv",
         auspice_config = "config/auspice_config_{subtype}.json",
         clades_file = "clade-labeling/{subtype}-clades.tsv",
         description = "config/description.md"
 
 files = rules.files.params
 
 def metadata_by_wildcards(wildcards):
-    if wildcards.subtype in ("h5n1", "h5nx"):
+    # H5 builds have extra clade-level metadata added to the metadata TSV.
+    # We may move this to a node-data JSON which would simplify the snakemake logic
+    # a bit -- see <https://github.com/nextstrain/avian-flu/issues/25>
+    if wildcards.subtype in ("h5n1", "h5nx", "cattle-flu-204"):

```

Comment:
This is a typo and means the "h5_label_clade" is missing from the metadata. It's also missing from the auspice-config for the h5n1-cattle-outbreak _genome_ build (which is the auspice-config we use here). Will fix.